### ISSUE TITLE
TypeScript typings for runtime

### ISSUE THREAD

**Issue by NekR** [NekR/offline-plugin#106]
There are problems with TypeScript and lack of typings for `offline-plugin/runtime`. Ideally, users still should be able to do this (which doesn't work right now in TypeScript):

``` ts
import { install as installOffline } from 'offline-plugin/runtime';

installOffline();
```

**Comment by seveves** [comment]
Looky looky! There is a PR! :robot:

**Comment by NekR** [comment]
Fixed in #196. Thank you @seveves :1st_place_medal:
